recipes/augustus/3.2.2/meta.yaml

```diff
@@ -0,0 +1,119 @@
+about:
+  home: http://bioinf.uni-greifswald.de/augustus/
+  license: ''
+  summary: 'AUGUSTUS is a gene prediction program for eukaryotes written by Mario
+    Stanke and Oliver Keller. It can be used as an ab initio program, which means
+    it bases its prediction purely on the sequence. AUGUSTUS may also incorporate
+    hints on the gene structure coming from extrinsic sources such as EST, MS/MS,
+    protein alignments and synthenic genomic alignments.'
+build:
+  number: 0
+package:
+  name: augustus
+  version: '3.2.2'
+  skip: True # [osx]
+requirements:
+  build:
+      - libgcc
+      - gcc
+      - bamtools
+      - perl-app-cpanminus
+      - perl-yaml
+      - perl-dbi
+      - boost
```

Comment:
Can you please pin your boost dependency like here: https://github.com/bioconda/bioconda-recipes/blob/28f764787310141bc74bf4ee90ace6432b9fa6b0/recipes/salmon/meta.yaml#L21
Please also not the build-string change.